src/libs/actions/IOU.js

```diff
@@ -697,7 +697,6 @@ function getSendMoneyParams(report, amount, currency, comment, paymentMethodType
         key: `${ONYXKEYS.COLLECTION.REPORT}${chatReport.reportID}`,
         value: {
             ...chatReport,
-            lastVisitedTimestamp: Date.now(),
```

Comment:
NAB - Why doesn't this get replaced by `lastReadTimestamp`?

Reply:
I think it's because `lastActionCreated` is what we need (present below), right @roryabraham ?

Reply:
I think this should be replaced by:

```js
lastReadTime: DateUtils.getDBTime(),
```